src/rivian/ble.py

```diff
@@ -0,0 +1,68 @@
+import platform
+import asyncio
+import secrets
+import logging
+
+from bleak import BleakClient, BLEDevice
+from rivian import utils
+
+_LOGGER = logging.getLogger(__name__)
+
+PHONE_ID_VEHICLE_ID_UUID = "AA49565A-4D4F-424B-4559-5F5752495445"
+PNONCE_VNONCE_UUID = "E020A15D-E730-4B2C-908B-51DAF9D41E19"
+BLE_ACTIVE_ENTRY_UUID = "5249565F-4D4F-424B-4559-5F5752495445"
+
+async def pair_phone(
+    device: BLEDevice, vehicle_id, phone_id, vehicle_key, private_key
+) -> bool:
+    success = True
```

Comment:
We should set this to False initially and then only set it to True if we get past the pair/bonding step.